src/persistentSubscription/utils/persistentSubscriptionSettings.ts

```diff
@@ -89,7 +89,7 @@ export interface PersistentSubscriptionSettings
 export interface PersistentSubscriptionToAllSettings
   extends PersistentSubscriptionSettingsGeneric {
   /**
-   * The exclusive position in the stream or transaction file the subscription should start from.
+   * The inclusive position in the stream or transaction file the subscription should start from.
```

Comment:
```suggestion
   * Where to start the subscription from. This can be from the start of the stream, from the end of the stream at the time of creation, or from an inclusive position in the stream.
```